### ISSUE TITLE
[Feature Request] add option for 'play random anime' to follow Prequel<->Sequel relations

### ISSUE THREAD

**Issue by Marius8** [erengy/taiga#1081]
I would like to have a setting added for the 'Play random anime' option to check if you have unwatched episodes of a Prequel.

So, if the option selects eg. the 4th season of SAO, but you still have unwatched episodes in the 1st season, that the feature plays those instead.

**Comment by erengy** [comment]
Makes sense, but Taiga isn't aware of prequel/sequel relationships, so this is currently not possible.

Taiga doesn't randomly play an episode from an anime that's not in your list, so I'm guessing S4 was in your plan-to-watch list even though you're currently watching S1?

**Comment by Marius8** [comment]
didn't happen to me with SAO, but yes, that's how it happened.
chose, SAO as an example, as it is a more common anime, in recent years
